config/routes.rb

```diff
@@ -0,0 +1,97 @@
+Rails.application.routes.draw do
+root to: 'media#index'
+  resources :albums
+  resources :movies
+  resources :books
+  resources :media, only: [:index, :show]
+  resources :books, :albums, :movies do
```

Comment:
Because of this line, the above `resources` lines are redundant and can be removed.